client/blocks/credit-card/payment-method.js

```diff
@@ -18,7 +18,7 @@ import { InlineCard, CardElements } from './elements';
  */
 
 export const getStripeCreditCardIcons = () => {
-	return Object.entries( getStripeServerData().icons ).map(
+	return Object.entries( getStripeServerData()?.icons ).map(
```

Comment:
Ah yeah, this should definitely be changed to

```js
return Object.entries( getStripeServerData()?.icons ?? {} ).map(
```

> I think like this particular line would result in an error with or without the optional chaining (and possibly a less helpful one this way)?

I think the error isn't helpful in any case to be honest 😅 

<img width="573" alt="image" src="https://user-images.githubusercontent.com/13835680/121271550-7904f000-c8b3-11eb-852d-4f5fe4a3211f.png">

And if there's an error there that crashes the rest of the Blocks support JS I think. I'd rather use the null coalescing and just add a `?? {}` so that the Stripe blocks support doesn't crash if we happen to forget to supply icons 😬

Reply:
Sorry; just to clarify my point with the error ^ not being useful is that when you don't see where exactly it's coming from — which we don't because webpack minifes the code during the build — that error isn't useful at all.

Reply:
Fixed in eea0676f5d0d0e6bebc7607c271c01ff74bd2d88

Reply:
Sounds good. And I really have no idea how that "like" snuck in to my comment there 😆